src/Dodo.HttpClient.ResiliencePolicies/HttpClientSettings.cs

```diff
@@ -8,16 +8,18 @@ public class HttpClientSettings
 	{
 		public TimeSpan HttpClientTimeout { get; }
```

Comment:
The most arguable thing here is should we really support both HttpClientTimeout and TimeoutOverall? Yes, they are techincally different things, but for the end-user prospective it looks like it could be the same thing. My http request will be completed or failed after N seconds. The reason in both cases - timeout. Maybe we should work with single timeout setting? What do you think?

Reply:
I agree and will prepare the change.